# HG changeset patch
# User Rik <rik@octave.org>
# Date 1515016167 28800
#      Wed Jan 03 13:49:27 2018 -0800
# Node ID 5553f0ada5c6ae61fa2448d311539e1a00b1ac91
# Parent  941ea3da921f98d5dddc9d24ba84ee9de3fafabc
Use American spelling of "color" in GUI preferences dialogs.

* settings-dialog.ui: Change British spelling "colour" to American spelling
"color" because Octave convention is for American spelling throughout code
base.

diff --git a/libgui/src/settings-dialog.ui b/libgui/src/settings-dialog.ui
--- a/libgui/src/settings-dialog.ui
+++ b/libgui/src/settings-dialog.ui
@@ -2819,29 +2819,29 @@
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="varedit_colors_box">
                <property name="title">
-                <string>Variable Editor Colours</string>
+                <string>Variable Editor Colors</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="varedit_alternate">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
-              <string>Use alternating row colours</string>
+              <string>Use alternating row colors</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
